docs/java/features/odata/overview.md

```diff
@@ -16,19 +16,124 @@ keywords:
 - generate
 ---
 
+## Supported OData versions
+| Version  | SAP Cloud SDK V3 |
+|----------|------------------|
+| OData v2 | &#10003;         |
+| OData v4 | &#10003;         |
+
+
 ## What is OData?
 
 [OData](https://www.odata.org/) is an open standard for building and consuming RESTful APIs. It defines a query language to send and retrieve data via HTTP and to perform operations on data.
 
-There are two widely used versions of the protocol: OData v2 and OData v4. The latter comes with more and improved functionality. Full details on the standards are to be found in the [documentation](https://www.odata.org/documentation/).
+There are two widely used versions of the protocol: OData v2 and OData v4. The latter comes with extended and improved functionality. You can find full details on OData standers in [documentation](https://www.odata.org/documentation/).
 
 ## What does the Cloud SDK offer?
 
-The Cloud SDK supports consumption of OData services through the following features:
+The Cloud SDK simplifies consumption of OData services through the following features:
+
+- We provide a type-safe OData client to build and execute OData requests
+- We simplify connectivity to SAP enabled OData services via type-safe abstractions over [Destinations](../connectivity/sdk-connectivity-destination-service), Authentication and some other related concepts.
+- A [code generator](../generate-typed-odata-v2-and-v4-client-for-java) to convert your OData service definition into type-safe Java client library.
+  - Service definition usually comes in form of `.edmx` file and contains metadata of Odata service. The simplest way to obtain it if you have access to the service is navigating to `http(s)://<service-path>/$metadata` endpoint. You can easily generate a client for any 3rd party service or a service that you've build on you own.
+  - You can invoke code generator via our command line interface (CLI) or via Maven plugin
+- For many popular SAP OData services we ship pre-generated type-safe clients. In SAP universe they are called Virtual Data Model or VDM. The most popular type-safe client module contains a collection of [all OData services of SAP S/4HANA Cloud](https://api.sap.com/package/SAPS4HANACloud?section=Artifacts)
+- We support both OData v2 and OData v4 protocols in the SAP Cloud SDK for Java.
+
+## OData features supported by SAP Cloud SDK for Java
+
+### Create, Read, Update, Delete - CRUD
```

Comment:
I like the improvement in structure. Looking at it again I would further improve by:

- Removing the Comment and Example query sections for now
- Linking the official OData 2.0 & 4.0 docu and
- Linking the Cloud SDK docu

I would do the last 2 things per section.

Also not all of the checkmarks are correct. Do you want to go over them together or should I just update the table?

Reply:
@MatKuhr 

This is exactly where I need help:

- Naming the features and agreeing on common names for them between Java & JS
- Putting correct checkmarks (I did it quick and dirty)
- I can have with linking of the Docu and official OData doc. That's also not where we have to do it in one go.